ios/FluentUI/Core/ControlHostingView.swift

```diff
@@ -36,7 +36,30 @@ open class ControlHostingContainer: NSObject {
 
         // Set the initial appearance of our control.
         self.updateRootView()
-        view.backgroundColor = UIColor.clear
+        self.configureHostedView()
+    }
+
+    required public init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    /// Adds `hostingController.view` to ourselves as a subview, and enables necessary constraints.
+    private func configureHostedView() {
+        guard let hostedView = hostingController.view else {
+            return
+        }
+        hostedView.backgroundColor = UIColor.clear
+
+        addSubview(hostedView)
+        hostedView.translatesAutoresizingMaskIntoConstraints = false
+
+        let requiredConstraints = [
```

Comment:
Have you tested scenarios where the SwiftUI View is set with the frame maxWidth/Height to .infinity and constraints are (or are not) set from the outer SwiftUI view?
i believe my main concern is about constraint priorities and how the SwiftUI View dictates the container UIView dimensions.

Reply:
That's a good question! I _believe_ that `AvatarGroup` should meet that requirement:

https://github.com/mischreiber/fluentui-apple/blob/6eca8a1ab1deb04c77a6bedee3436338e107e43a/ios/FluentUI/AvatarGroup/AvatarGroup.swift#L197-L200

And the layout/overflow behavior matches the existing demo app:

https://user-images.githubusercontent.com/4934719/159806575-1bee69e8-3e32-4297-9c3a-e602463c0485.mov